components/plugin_marketplace/marketplace_modal.tsx

```diff
@@ -84,81 +90,89 @@ export const InstalledPlugins = ({installedPlugins, changeTab}) => {
     return <MarketplaceList plugins={installedPlugins}/>;
 };
 
-InstalledPlugins.propTypes = {
-    installedPlugins: PropTypes.array.isRequired,
-    changeTab: PropTypes.func,
+export type MarketplaceModalProps = {
+    show: boolean;
```

Comment:
In the propTypes, show is not marked as required. Not sure if it should be required here.

Reply:
`FullScreenModal` expects a `boolean` and given that `isModalOpen` always return one, it's fine to require it here.